src/Ethereum/ABI/ParamFactory.h

```diff
@@ -14,6 +14,8 @@
 #include <string>
 #include <vector>
 
+#include "Wasm.h"
```

Comment:
Is this really needed here?

Reply:
It fails to build without some int defines